README.md

```diff
@@ -33,7 +33,7 @@ First, install Node.js 14+.
 
 Next, install dependencies by running `npm install`.
 
-Install NGINX. Depending on your operating system and how you install NGINX, you may need to change the absolute paths in the development [`nginx.conf`](/nginx.conf).
+Install NGINX. Depending on your operating system and how you install NGINX.
```

Comment:
Maybe we can now remove the second sentence entirely?

Reply:
:grin:  yes, it makes no sense now!